posthog/api/test/notebooks/__snapshots__/test_notebook.ambr

```diff
@@ -0,0 +1,709 @@
+         "posthog_team"."autocapture_exceptions_opt_in",
+         "posthog_team"."autocapture_exceptions_errors_to_ignore",
+         "posthog_team"."session_recording_opt_in",
+         "posthog_team"."capture_console_log_opt_in",
+         "posthog_team"."capture_performance_opt_in",
+         "posthog_team"."session_recording_version",
+         "posthog_team"."signup_token",
+         "posthog_team"."is_demo",
+         "posthog_team"."access_control",
+         "posthog_team"."inject_web_apps",
+         "posthog_team"."test_account_filters",
+         "posthog_team"."test_account_filters_default_checked",
+         "posthog_team"."path_cleaning_filters",
+         "posthog_team"."timezone",
+         "posthog_team"."data_attributes",
+         "posthog_team"."person_display_name_properties",
+         "posthog_team"."live_events_columns",
+         "posthog_team"."recording_domains",
+         "posthog_team"."primary_dashboard_id",
+         "posthog_team"."extra_settings",
+         "posthog_team"."correlation_config",
+         "posthog_team"."session_recording_retention_period_days"
+  FROM "posthog_team"
+  WHERE "posthog_team"."id" = 2
+  LIMIT 21 /*controller='project_notebooks-list',route='api/projects/%28%3FP%3Cparent_lookup_team_id%3E%5B%5E/.%5D%2B%29/notebooks/%3F%24'*/
+  '
+---
+# name: TestNotebooks.test_updates_notebook.10
+  '
+  SELECT "posthog_notebook"."id",
+         "posthog_notebook"."short_id",
+         "posthog_notebook"."team_id",
+         "posthog_notebook"."title",
+         "posthog_notebook"."content",
+         "posthog_notebook"."deleted",
+         "posthog_notebook"."version",
+         "posthog_notebook"."created_at",
+         "posthog_notebook"."created_by_id",
+         "posthog_notebook"."last_modified_at",
+         "posthog_notebook"."last_modified_by_id"
+  FROM "posthog_notebook"
+  WHERE "posthog_notebook"."id" = '01891c84-73f6-0000-8ea4-5d223f446585'::uuid
+  LIMIT 21 /*controller='project_notebooks-detail',route='api/projects/%28%3FP%3Cparent_lookup_team_id%3E%5B%5E/.%5D%2B%29/notebooks/%28%3FP%3Cshort_id%3E%5B%5E/.%5D%2B%29/%3F%24'*/
+  '
+---
+# name: TestNotebooks.test_updates_notebook.11
+  '
+  SELECT "posthog_notebook"."id",
+         "posthog_notebook"."short_id",
+         "posthog_notebook"."team_id",
+         "posthog_notebook"."title",
+         "posthog_notebook"."content",
+         "posthog_notebook"."deleted",
+         "posthog_notebook"."version",
+         "posthog_notebook"."created_at",
+         "posthog_notebook"."created_by_id",
+         "posthog_notebook"."last_modified_at",
+         "posthog_notebook"."last_modified_by_id"
+  FROM "posthog_notebook"
+  WHERE "posthog_notebook"."id" = '01891c84-73f6-0000-8ea4-5d223f446585'::uuid
+  LIMIT 21
+  FOR
+  UPDATE /*controller='project_notebooks-detail',route='api/projects/%28%3FP%3Cparent_lookup_team_id%3E%5B%5E/.%5D%2B%29/notebooks/%28%3FP%3Cshort_id%3E%5B%5E/.%5D%2B%29/%3F%24'*/
```

Comment:
cool the snapshot captured the fir update, yay

So, if someone breaks it in future, this will highlight it to them